ingestion/src/metadata/profiler/source/metadata_ext.py

```diff
@@ -226,36 +225,3 @@ def get_database_names(self) -> Iterable[str]:
         except Exception as exc:
             logger.debug(f"Failed to fetch database names {exc}")
             logger.debug(traceback.format_exc())
-
-    def get_table_entities(self, database):
```

Comment:
I don't think it was used at all @ulixius9, I only see in `_iter` the call to the ES search